docs/ocf.md

```diff
@@ -0,0 +1,107 @@
+ZJS API for OCF
+==================
+
+* [Introduction](#introduction)
+* [Web IDL](#web-idl)
+* [API Documentation](#api-documentation)
+* [Sample Apps](#sample-apps)
+
+Introduction
+------------
+ZJS provides OCF Server API's which allow communication using the OCF networking
+protocol.
+
+Web IDL
+-------
+This IDL provides an overview of the interface; see below for documentation of
+specific API functions.
+
+```javascript
+// require returns an OCFObject
+// var ocf = require('ocf');
+
+dictionary ResourceInit {
+    string resourcePath;      // OCF resource path
+    string[] resourceTypes;   // List of resource types
+    string[] interfaces;      // List of interfaces for resource types
+    boolean discoverable;     // Is resource discoverable
+    boolean observable;       // Is resource observable
+    boolean secure;           // Is resource security enabled
+    boolean slow;             // Is resource a slow reader
+    object properties;        // Dictionary of resource properties
+};
+
+interface OCFResource {
+    string resourcePath;      // Path for this resource
+    object properties;        // Application specific resource properties
+};
+
+interface OCFRequest {
+    OCFResource target;       // Target/destination resource
+    OCFResource source;       // Source/origin resource
+    Promise<void> respond(object data);
+};
+
+// OCFServer is an EventEmitter
+interface OCFServer {
+    Promise<OCFResource> register(ResourceInit init);
+};
+
+interface OCFObject {
+    OCFServer server;         // OCF server object
+};
+
+// listener for onretrieve event
+callback RetrieveCallback = void (OCFRequest request, boolean observe);
+// listener for onupdate event
+callback UpdateCallback = void (OCFRequest request);
+
+```
+
```

Comment:
I wonder if you could use the definitions in 
https://github.com/01org/iot-js-api/tree/master/api/ocf
that is, aligned with iotivity-node?